docs/docs/cmd/graph/subscription/subscription-add.mdx

```diff
@@ -47,6 +47,10 @@ To subscribe to change notifications of Outlook contacts, events, or messages in
 - Use the corresponding application permission to subscribe to changes of items in a folder or mailbox of any user in the tenant.
 - Do not use the Outlook sharing permissions (Contacts.Read.Shared, Calendars.Read.Shared, Mail.Read.Shared, and their read/write counterparts), as they do not support subscribing to change notifications on items in shared or delegated folders.
 
+If you want receive notifications through Azure Event Hub, the `notificationUrl` must point to your Event Hubs location.
+
+If you want receive notifications through Azure Event Grid, the `notificationUrl` must point to your Event Grid Partner Topic.
```

Comment:
To be honest, I don't know if it's so useful to mention this. Isn't this obvious?

Reply:
It should be obvious and of course, details are described in the Graph API doc. I would expect that users will be familiar with it.